game_eggs/eft/egg-singleplayer-tarkov

```diff
@@ -0,0 +1,62 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-02-21T03:47:21-05:00",
+    "name": "Singleplayer Tarkov",
+    "author": "dino@papertots.page",
+    "description": "The Stay in Tarkov project was born due to Battlestate Games' (BSG) reluctance to create the pure PvE version of Escape from Tarkov. The project's aim is simple, create a Cooperation PvE experience that retains progression.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/pterodactyl\/yolks:debian": "ghcr.io\/pterodactyl\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": ".\/Aki.Server.exe",
```

Comment:
Running exe files with no wine? That will not work? Have you even ever tested this?

Reply:
It works somehow, and I can connect to the tarkov server running on Pterodactyl on my local server. I even started a game and there was no error messages in the console.

Reply:
I see, they force the .exe extention even on linux